src/backend/cgcod.c

```diff
@@ -250,7 +250,17 @@ void codgen()
             block* b = dfo[dfoidx];
             blcodgen(b);                        // gen code in depth-first order
             //printf("b->Bregcon.used = %s\n", regm_str(b->Bregcon.used));
-            cgreg_used(dfoidx,b->Bregcon.used); // gather register used information
+            regm_t used = b->Bregcon.used;
+            if (config.ehmethod == EH_DWARF)
+                switch (b->BC)
+                {
+                    case BCjcatch:
+                    case BC_finally:
+                    case BC_lpad:
+                        used |= allregs;        // can't enregister live variables on landing pads
```

Comment:
It means that any variable that is 'live' on a landing pad will not get enregistered.